modules/steam/steam.nix

```diff
@@ -146,6 +148,34 @@ let
 
     set -e
 
+    # OOBE handling.
+    # (See steam-jupiter.sh in steam-jupiter-oobe and steam-jupiter-stable)
+    # On first boot, we need to start whatever steam we have packaged, to
+    # allow the user to connect to the internet. We don't have the luxury
+    # that Steam's OOBE has to be heavy-handed. So instead we'll see if it
+    # looks like the user has never started steam. If it looks that way,
+    # we'll init OOBE, and otherwise undo it.
+    (
+      STEAM_LINKS="$HOME"/.steam
+      STEAM_DIR="$HOME"/.local/share/Steam
+      REGISTRY="$STEAM_LINKS"/registry.vdf
+
+      echo ":: Checking if we're online, or in OOBE..."
```

Comment:
```suggestion
      echo ":: Checking if we're not online, or in OOBE..."
```

Reply:
I'll use `offline`, to reduce chances of double-negatives.